src/theme.js

```diff
@@ -1,207 +1,212 @@
+      "tab.unfocusedActiveBorder"   : color.bg.canvas,
+      "tab.activeBorderTop"         : "#f9826c",
+
+      "breadcrumb.foreground"               : color.text.tertiary,
+      "breadcrumb.focusForeground"          : color.text.primary,
+      "breadcrumb.activeSelectionForeground": color.text.secondary,
+      "breadcrumbPicker.background"         : color.bg.overlay,
+
+      "editor.foreground"                 : editorForeground,
+      "editor.background"                 : color.bg.canvas,
+      "editorWidget.background"           : color.bg.overlay,
+      "editor.foldBackground"             : color.bg.canvasInset,
+      "editor.lineHighlightBackground"    : color.codemirror.activelineBg,
+      "editorLineNumber.foreground"       : color.codemirror.linenumberText,
+      "editorLineNumber.activeForeground" : color.text.primary,
+      "editorIndentGuide.background"      : color.border.secondary,
+      "editorIndentGuide.activeBackground": color.border.primary,
+      "editorWhitespace.foreground"       : color.border.tertiary,
+      "editorCursor.foreground"           : themes({ light: scale.blue[7], dark: scale.blue[2], dimmed: scale.blue[2] }),
+
+      "editor.findMatchBackground"          : themes({ light: scale.yellow[4], dark: "#ffd33d44", dimmed: "#ffd33d44" }),
+      "editor.findMatchHighlightBackground" : themes({ light: "#ffdf5d66", dark: "#ffd33d22", dimmed: "#ffd33d22" }),
+      "editor.inactiveSelectionBackground"  : themes({ light: "#0366d611", dark: "#3392FF22", dimmed: "#3392FF22" }),
+      "editor.selectionBackground"          : themes({ light: "#0366d625", dark: "#3392FF44", dimmed: "#3392FF44" }),
+      "editor.selectionHighlightBackground" : themes({ light: "#34d05840", dark: "#17E5E633", dimmed: "#17E5E633" }),
+      "editor.selectionHighlightBorder"     : themes({ light: "#34d05800", dark: "#17E5E600", dimmed: "#17E5E600" }),
+      "editor.wordHighlightBackground"      : themes({ light: "#34d05800", dark: "#17E5E600", dimmed: "#17E5E600" }),
+      "editor.wordHighlightStrongBackground": themes({ light: "#34d05800", dark: "#17E5E600", dimmed: "#17E5E600" }),
+      "editor.wordHighlightBorder"          : themes({ light: "#24943e99", dark: "#17E5E699", dimmed: "#17E5E699" }),
+      "editor.wordHighlightStrongBorder"    : themes({ light: "#24943e50", dark: "#17E5E666", dimmed: "#17E5E666" }),
+      "editorBracketMatch.background"       : themes({ light: "#34d05840", dark: "#17E5E650", dimmed: "#17E5E650" }),
+      "editorBracketMatch.border"           : themes({ light: "#34d05800", dark: "#17E5E600", dimmed: "#17E5E600" }),
+
+      "editorGutter.modifiedBackground": color.diff.change.border,
+      "editorGutter.addedBackground"   : color.diff.addition.border,
+      "editorGutter.deletedBackground" : color.diff.deletion.border,
+
+      "diffEditor.insertedTextBackground": color.diff.addition.bg,
+      "diffEditor.removedTextBackground":  color.diff.deletion.bg,
```

Comment:
These colors are stored as `rgba()` in the dark themes in @primer/primitives, but VSCode needs them to be in `#RRGGBBAA` form here. My VSCode defaults them to a very bright and opaque red or green on the dark modes.

Reply:
👍  Yes, seems VS Code [only allows](https://code.visualstudio.com/api/references/theme-color#color-formats) the `#RRGGBBAA` format for transparency.

Hmm.. seems that `rgba()` has slightly better browser support, so we might still wanna keep using it for `@primer/primitives`. For the VSCode theme we could try to convert it to `#RRGGBBAA`.

Reply:
@kherock thanks for reporting this! Was just about to report it myself :)

Reply:
Ok, it's currently fixed by wrapping the `@primer/primitives` color with a [`chroma(...).hex()`](https://gka.github.io/chroma.js/#color-hex) that supports `#RRGGBBAA`:

![Screen Shot 2021-01-05 at 16 38 27](https://user-images.githubusercontent.com/378023/103619127-84569700-4f74-11eb-89f9-4617849c18ec.png)

But it could break anytime if `@primer/primitives` uses `rgba()` for any of the other colors. Would be better if we could convert all colors before generating the theme.

Reply:
I'm not entirely sure if this Is the same issue, but the same thing occurs on the git popup in the editor. 
<img width="330" alt="Screen Shot 2021-01-06 at 1 07 59 PM" src="https://user-images.githubusercontent.com/18719073/103804561-5333a080-5020-11eb-9a3d-936e576466f8.png">

Reply:
@jeffgennari Yes.. thanks for spotting this. Should be fixed with https://github.com/primer/github-vscode-theme/pull/98/commits/66bc8f0f9430b44e40da279f5c4fdd3d0c3b584a.

![Screen Shot 2021-01-07 at 11 27 26](https://user-images.githubusercontent.com/378023/103843636-85ee9f00-50db-11eb-9114-92431c1ef7a0.png)